xml/System.Collections.Immutable/ImmutableList`1.xml

```diff
@@ -1883,6 +1883,7 @@
  ]]></format>
         </remarks>
         <exception cref="T:System.NotImplementedException" />
+        <exception cref="T:System.NotSupportedException">Always thrown.</exception>
```

Comment:
This one is weird, because the previous line indicates a `NotImplementedException` already being thrown. Can you please verify, @eiriktsarpalis / @layomia ?

Reply:
This is what exists in the method implementation https://github.com/dotnet/runtime/blob/dfb3022b59700f33ff8f7034982e5ce642d80688/src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableList_1.cs#L940

Reply:
I think the tool should have replaced the previous line?

Reply:
No, the tool does not replace anything pre-existing in the xml. I'm not sure why `NotImplementedException` is there, so I'll remove it.

Reply:
Thanks for checking, I was having trouble finding that file.